nservicebus/hosting/azure-functions-service-bus/in-process/index_configuration_ASBFunctions_[4.2,].partial.md

```diff
@@ -0,0 +1,35 @@
+`ServiceBusTriggeredEndpointConfiguration` loads certain configuration values from the Azure Function host environment in the following order:
```

Comment:
note that this needs to be changed to next version (4.3?) once released, rather than the current version break on 4.2